### PR TITLE
only yoni and yoav

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/SetScoringMode.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    private ScoringMode scoringMode;
+
+    protected SetScoringMode(ScoringMode mode) {
+        scoringMode = mode;
+    }
+
+    @Override
+    public void initialize() {
+        System.out.println(scoringMode);
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/SetScoringMode.java:20]: remove sout

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 2]
```diff
@@ -53,7 +55,9 @@ public SmartJoystick getSecondJoystick() {
     }
 
     public void initButtons() {
-        secondJoystick.A.whileTrue(new MoveElbowAndWrist(Rotation2d.fromDegrees(0),Rotation2d.fromDegrees(0)));
+        secondJoystick.A.onTrue(new MoveElbowAndWrist(Rotation2d.fromDegrees(90),Rotation2d.fromDegrees(0)));
+        secondJoystick.B.onTrue(new SwitchBetweenScoringModes());
+        secondJoystick.X.onTrue(new MoveShooterToAngle(Rotation2d.fromDegrees(20)));
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/OI.java:60]: delete

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/SimulationElbow/SimulationElbow.java [hunk 3]
```diff
@@ -74,7 +74,9 @@ public void updateInputs(ElbowInputsAutoLogged inputs) {
 
         inputs.appliedOutput = appliedVoltage;
         inputs.outputCurrent = elbowSimulation.getCurrentDrawAmps();
-        inputs.position = elbowSimulation.getAngleRads() + SimulationElbowConstants.SIMULATION_OFFSET;
+
+
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/SimulationElbow/SimulationElbow.java:78]: remove

File: src/main/java/edu/greenblitz/robotName/commands/intake/NoteToIntake.java [hunk 4]
```diff
@@ -1,10 +1,5 @@
-
 package edu.greenblitz.robotName.commands.intake;
 
-import edu.greenblitz.robotName.subsystems.Intake.IntakeConstants;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-
-
 public class NoteToIntake extends IntakeCommand {
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/intake/NoteToIntake.java:5]: bruh what is this

File: src/main/java/edu/greenblitz/robotName/commands/intake/NoteToShooter.java [hunk 5]
```diff
@@ -3,7 +3,6 @@
 import edu.greenblitz.robotName.subsystems.Intake.Intake;
 import edu.greenblitz.robotName.subsystems.shooter.Funnel.Funnel;
 import edu.greenblitz.robotName.utils.GBCommand;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/intake/NoteToShooter.java:6]: no

File: src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java [hunk 6]
```diff
@@ -1,7 +1,5 @@
 package edu.greenblitz.robotName.commands.intake;
 
-import edu.greenblitz.robotName.subsystems.Intake.IntakeConstants;
-
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java:4]: no

**Pull request closed without merging**